dashboard/config/linux/bits/base.yml

```diff
@@ -126,7 +127,8 @@ config:
  # (!ARCH_WANTS_NO_INSTR [=y] || STACK_VALIDATION [=n] || GCC_VERSION [=110200]>=120000 || CLANG_VERSION [=0]>=130000)
  - KCOV: [-arm, -s390, -nokcov]
  - KCOV_INSTRUMENT_ALL: [-arm, -s390, -nokcov]
- - KCOV_ENABLE_COMPARISONS: [-arm, -s390, -nokcov]
+ # Doesn't boot with KCOV_ENABLE_COMPARISONS on Cuttlefish.
+ - KCOV_ENABLE_COMPARISONS: [-arm, -s390, -nokcov, -cuttlefish]
```

Comment:
Do you plan to re-enable it soon?

Reply:
Yes; I'll create a task to debug this. Just as a question about priority, based on the documentation `KCOV_ENABLE_COMPARISONS` 
< Enables comparison operands collection by KCOV

Do you know if this just expands the coverage values we can collect?

Reply:
It extracts "magic numbers" from the code.
I think it is a good source of coverage, but don't have exact number.
CC @dvyukov